examples/config/example.py

```diff
@@ -0,0 +1,46 @@
+from confu.config import Config
+from confu.schema import Int, List, Schema, Str
+
+
+# declaring schema classes
+class Nested_Schema(Schema):
+    int_attr = Int("int_attr", default=10)
```

Comment:
The name attribute is optional and will default to whatever the property definition name is, providing it used to be mandatory but now it only makes sense if you want the field name to differ in the schema from the definition name here.

e.g., that line should be just

```
int_attr = Int(default=10)
```

@jason20c